packages/suite/src/components/suite/modals/ReduxModal/DeviceContextModal/DevicePromptModal.tsx

```diff
@@ -76,10 +75,10 @@ const DevicePromptModalRenderer = ({
                             deviceModelInternal={deviceModelInternal}
                             deviceUnitColor={device?.features?.unit_color}
                             isConfirmed={isConfirmed}
+                            onCancel={isActionAbortable ? onAbort : undefined}
                         />
                     )
                 }
-                headerComponent={isActionAbortable ? <AbortButton onAbort={onAbort} /> : undefined}
```

Comment:
I see similar pattern during onboarding
![93930FB7-9129-4E28-B0AD-EE33C5BBA118](https://github.com/trezor/trezor-suite/assets/66002635/fc46b65c-5202-4d63-a8e0-21c50f47ff3d)

Reply:
This change was not in the scope of the issue. It might be better but I am not sure and did not want to solve that. Also, it would be nice to either remove the `AbortButton` completely or keep it there. It's currently still used in 1 place i believe.

To make it quick, please make it tertiary and we will solve the rest later :)

Reply:
a3e4aab07883b154062ee7c60b8c33b64d381013